book/chapter.css

```diff
@@ -189,6 +189,7 @@ table tr:last-child td {
   fill: none;
   stroke: #ccc;
   stroke-width: 2px;
+  /*marker-end: url(#arrowhead);*/
```

Comment:
I was trying to add arrowheads to the svg-based tree diagrams but couldn't quite get them to point the right way on bezier curves. Leaving my WIP code in as I hope to fix the bezier curve issue soon.